GliaWidgets/Sources/Theme/Survey/Theme.Survey.swift

```diff
@@ -1,32 +1,50 @@
-import Foundation
 import UIKit
 
 extension Theme {
     public struct SurveyStyle {
         /// Layer style.
         public var layer: Layer
+
         /// Header text style.
         public var title: Text
+
         /// Submit button style.
         public var submitButton: Button
-        /// Cancell button style.
-        public var cancellButton: Button
+
+        /// Cancel button style.
+        public var cancelButton: Button
```

Comment:
I'm not sure anyone use it directly, however, it still public interface and should we deprecate it, create proper var name without type? @gersonnoboa  @igorkravchenko wdyt?

Reply:
Added deprecated `cancellButton` that will use the new `cancelButton` under the hood. [48d6b41](https://github.com/salemove/ios-sdk-widgets/pull/887/commits/48d6b41097c2c82213033740039e8f97811a445e)

Reply:
@yurii-glia I think it makes sense to deprecate it and provide proper name,  otherwise it is potentially a breaking change.

> proper var name without type
What do you mean by "without type"? Without typo?

Reply:
yes, "without typo")